lib/capybara/node/finders.rb

```diff
@@ -219,28 +216,28 @@ def find_by_id(id, **options, &optional_filter_block)
       #     page.all('a', text: 'Home')
       #     page.all('#menu li', visible: true)
       #
-      # By default Capybara's waiting behavior will wait up to `Capybara.default_max_wait_time`
+      # By default Capybara's waiting behavior will wait up to {Capybara.configure default_max_wait_time}
       # seconds for matching elements to be available and then return an empty result if none
       # are available. It is possible to set expectations on the number of results located and
       # Capybara will raise an exception if the number of elements located don't satisfy the
-      # specified conditions.  The expectations can be set using
+      # specified conditions. The expectations can be set using:
       #
       #     page.assert_selector('p#foo', count: 4)
       #     page.assert_selector('p#foo', maximum: 10)
       #     page.assert_selector('p#foo', minimum: 1)
       #     page.assert_selector('p#foo', between: 1..10)
       #
-      # See {Capybara::Helpers#matches_count?} for additional information about
+      # See {Capybara::Result#matches_count?} for additional information about
```

Comment:
[question] I don't know where is this "additional information"? 

- `Helpers` has no `#matches_count?`.
  - https://www.rubydoc.info/gems/capybara/3.21.0/Capybara/Helpers
- `BaseQuery` has `#matches_count?` but it's a private doc.
  - https://www.rubydoc.info/gems/capybara/3.21.0/Capybara/Queries/BaseQuery#matches_count%3F-instance_method
- `Result` has `#matches_count?` but it has no doc.
  - https://www.rubydoc.info/gems/capybara/3.21.0/Capybara/Result#matches_count%3F-instance_method

Reply:
It used to be in Helpers but was moved out some time ago.  It's fine to just remove the line - the options are pretty self-explanatory

Reply:
Thanks, I removed it via 273e2547f5a67355c4ef7373276abac19930dcef